@aescolar Regarding the following documentation build failure, you can cherry-pick https://github.com/zephyrproject-rtos/zephyr/pull/56160/commits/6c0b69559bb8edab3be31b1f46f861e23b04ebc8 to fix the issue.

```
make: Entering directory '/home/runner/work/zephyr/zephyr/doc'
make html DT_TURBO_MODE=1
make[1]: Entering directory '/home/runner/work/zephyr/zephyr/doc'
cmake \
	-GNinja \
	-B_build \
	-S. \
	-DDOC_TAG=development \
	-DSPHINXOPTS="-q -W -t publish" \
	-DLATEXMKOPTS="-halt-on-error -no-shell-escape" \
	-DDT_TURBO_MODE=1
Loading Zephyr module(s) (Zephyr repository): doc
-- Found Python3: /usr/bin/python3.10 (found suitable exact version "3.10.6") found components: Interpreter 
-- Cache files will be written to: /home/runner/.cache/zephyr
-- Found west (found suitable version "1.0.0", minimum required is "1.0.0")
CMake Error at /home/runner/work/zephyr/zephyr/cmake/modules/zephyr_module.cmake:77 (message):
  Traceback (most recent call last):

    File "/home/runner/work/zephyr/zephyr/scripts/zephyr_module.py", line 718, in <module>
      main()
    File "/home/runner/work/zephyr/zephyr/scripts/zephyr_module.py", line 663, in main
      west_projs = west_projects()
    File "/home/runner/work/zephyr/zephyr/scripts/zephyr_module.py", line 531, in west_projects
      manifest = Manifest.from_file()
    File "/home/runner/.local/lib/python3.10/site-packages/west/manifest.py", line 1170, in from_file
      return Manifest(topdir=topdir, config=config,
    File "/home/runner/.local/lib/python3.10/site-packages/west/manifest.py", line 1396, in __init__
      self._load_validated()
    File "/home/runner/.local/lib/python3.10/site-packages/west/manifest.py", line 1791, in _load_validated
      self._load_projects(manifest_data, url_bases, defaults)
    File "/home/runner/.local/lib/python3.10/site-packages/west/manifest.py", line [21](https://github.com/zephyrproject-rtos/zephyr/actions/runs/4502917193/jobs/7925415081?pr=55696#step:7:22)14, in _load_projects
      self._import_from_project(project, imp)
    File "/home/runner/.local/lib/python3.10/site-packages/west/manifest.py", line 2300, in _import_from_project
      self._import_map_from_project(project, imp)
    File "/home/runner/.local/lib/python3.10/site-packages/west/manifest.py", line 2327, in _import_map_from_project
      imported = self._import_content_from_project(project, imap.file)
    File "/home/runner/.local/lib/python3.10/site-packages/west/manifest.py", line 2395, in _import_content_from_project
      content = self._ctx.project_importer(project, path)
    File "/home/runner/.local/lib/python3.10/site-packages/west/manifest.py", line 164, in _default_importer
      raise ManifestImportFailed(project, file)

  west.manifest.ManifestImportFailed: ManifestImportFailed: project <Project
  bsim ('/home/runner/work/zephyr/tools/bsim') at
  bc[22](https://github.com/zephyrproject-rtos/zephyr/actions/runs/4502917193/jobs/7925415081?pr=55696#step:7:23)891208d453a8cb851e4ecdc8f165f3a33890> value west.yml

Call Stack (most recent call first):
  /home/runner/work/zephyr/zephyr/cmake/modules/doc.cmake:24 (include)
  /home/runner/work/zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:75 (include)
  /home/runner/work/zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:117 (include_boilerplate)
  CMakeLists.txt:7 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/runner/work/zephyr/zephyr/doc/_build/CMakeFiles/CMakeOutput.log".
make[1]: *** [Makefile:[23](https://github.com/zephyrproject-rtos/zephyr/actions/runs/4502917193/jobs/7925415081?pr=55696#step:7:24): configure] Error 1
make[1]: Leaving directory '/home/runner/work/zephyr/zephyr/doc'
make: Leaving directory '/home/runner/work/zephyr/zephyr/doc'
make: *** [Makefile:17: html-fast] Error 2
```

--- a/.github/workflows/doc-build.yml
+++ b/.github/workflows/doc-build.yml
@@ -69,6 +69,7 @@ jobs:
     - name: west setup
       run: |
         west init -l .
+        west update bsim
 
     - name: build-docs
       run: |
@@ -154,6 +155,7 @@ jobs:
     - name: west setup
       run: |
         west init -l .
+        west update bsim
 
     - name: build-docs
       run: |

--- a/west.yml
+++ b/west.yml
@@ -22,9 +22,16 @@ manifest:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
 
+  group-filter: [-babblesim]
+
   #
   # Please add items below based on alphabetical order
   projects:
+    - name: bsim
+      repo-path: babblesim-manifest
+      revision: 908ffde6298a937c6549dbfa843a62caab26bfc5
+      import:
+        path-prefix: tools
     - name: canopennode
       revision: dec12fa3f0d790cafa8414a4c2930ea71ab72ffd
       path: modules/lib/canopennode

